manifests/foreman_proxy_content.pp

```diff
@@ -9,27 +9,31 @@
 # $foreman_proxy_fqdn::             FQDN of the foreman proxy
 #                                   type:String
 #
+# $foreman_proxy_cname::            additional names of the foreman proxy
+#                                   type:Array
+#
 # $certs_tar::                      Path to tar file with certs to generate
 #                                   type:Optional[Stdlib::Absolutepath]
 #
 class certs::foreman_proxy_content (
-  $parent_fqdn        = $fqdn,
-  $foreman_proxy_fqdn = $certs::node_fqdn,
-  $certs_tar          = $certs::params::certs_tar
+  $parent_fqdn          = $fqdn,
+  $foreman_proxy_fqdn   = $certs::node_fqdn,
+  $foreman_proxy_cname  = $certs::cname,
```

Comment:
Maybe `$certs` name space isn't loaded when this class is processed in foreman-proxy-certs-generate?